src/ui/views/Dashboard/Header.tsx

```diff
@@ -247,6 +249,7 @@ const Header = ({ loading }) => {
     // usewallet.checkUserDomain(wallet.username);
   };
   const switchAccount = async (account) => {
+    setSwitchLoading(true);
```

Comment:
This should have a try catch finally around the statements in this function so that setSwitchLoading(false) is always called